.travis.yml

```diff
@@ -1,8 +1,16 @@
-language: node_js
+language:       node_js
 node_js:
   - "6"
   - "7"
-cache: yarn
+cache:          yarn
+before_install:
+  - echo -e "repo_token:\n  $COVERALLS_REPO_TOKEN" >> ./.coveralls.yml
+
 script:
   - yarn lint
   - yarn test
+  - yarn coverage
+  - yarn report-coverage
```

Comment:
Do these two lines need to be inside both places? I don't think so. Because of this, coveralls comments the status twice. Try removing these from here and let's keep the ones inside `after-script`.